_docs-sources/patcher/running-patcher/update.md

```diff
@@ -0,0 +1,88 @@
+# Patcher Update
+
+The Patcher update command allows you to update some or all of the module dependencies in the current folder and any child folders.
```

Comment:
```suggestion
The `patcher update` command allows you to update some or all of the module dependencies in the current folder and any child folders.
```

Reply:
I originally put `patcher update` here but then removed it as there are example usages in the subsections below